File: libs/sdk-dotnet/Models/SessionExecuteResponse.cs
// Copyright Daytona Platforms Inc.
// SPDX-License-Identifier: Apache-2.0

using System.Text.Json.Serialization;

namespace Daytona.Sdk.Models;

public record SessionExecuteResponse
{
    [JsonPropertyName("cmdId")] public string? CmdId { get; init; }
    [JsonPropertyName("output")] public string? Output { get; init; }
    [JsonPropertyName("exitCode")] public int? ExitCode { get; init; }

    public string? Stdout { get; init; }
    public string? Stderr { get; init; }
}

File: libs/sdk-java/src/main/java/io/daytona/sdk/model/SessionExecuteResponse.java
// Copyright Daytona Platforms Inc.
// SPDX-License-Identifier: Apache-2.0

package io.daytona.sdk.model;

public class SessionExecuteResponse extends io.daytona.toolbox.client.model.SessionExecuteResponse {
    public SessionExecuteResponse() {}

    public SessionExecuteResponse(io.daytona.toolbox.client.model.SessionExecuteResponse source) {
        super();
        if (source != null) {
            setCmdId(source.getCmdId());
            setOutput(source.getOutput());
            setExitCode(source.getExitCode());
        }
    }
}
